content/ADR-74-add-emote-schema.md

```diff
@@ -140,6 +140,18 @@ Since the ADR74 Emotes will have proper categories now, we need to start saving
 In summary, for emotes, the metadata stored in the contract will now be:
 `${version}:${type}:${name}:${description}:${category}:${bodyShapeTypes}:${loop}`
 
+##### 6. Extend Collection contract metadata to store sound and geometry properties
+As of *Emotes 2.0* initiative, emotes will now have the possiblity to have additional properties to enhance their behavior. This new properties are *sound* and *geometry*. To categorize this, a new section will be added to the end of the current emote metadata stored in the contract.
+
+The possible values will be:
+`s` : Emote has only sound
+`g` : Emote has only additional geometry
+`sg`: Emote has sound and geometry
+` ` : Emote has no sound and no additional geometry
```

Comment:
Is this a whitespace or just nothing?

Reply:
I would remove the example of no sound and no geometry as I assume it would be the same as how it was before